packages/squiggle-lang/src/fr/scale.ts

```diff
@@ -20,13 +21,23 @@ const commonRecord = frRecord(
   ["tickFormat", frOptional(frString)]
 );
 
+function checkMinMax(min: number | null, max: number | null) {
+  if (min !== null && max !== null && max <= min) {
```

Comment:
I assume we should also have an error if min=max. Not sure what we should show then.

Reply:
This code throws if min=max.